Modules/IO/NIFTI/include/itkNiftiImageIO.h

```diff
@@ -238,6 +269,10 @@ class ITKIONIFTI_EXPORT NiftiImageIO : public ImageIOBase
   double m_RescaleSlope{ 1.0 };
   double m_RescaleIntercept{ 0.0 };
 
+  bool m_ConvertRAS{ false };
+  bool m_ConvertRASVectors{ false };
+  bool m_ConvertRASDisplacementVectors{ true };
```

Comment:
No serious flaws found so far 😃 But I'm wondering, could we do without `m_ConvertRAS`? It seems to me that it isn't really a _property_ like m_ConvertRASVectors and m_ConvertRASDisplacementVectors. It should always just be equal to:

`(m_ConvertRASVectors && intent_code == NIFTI_INTENT_VECTOR) || (m_ConvertRASDisplacementVectors && intent_code == NIFTI_INTENT_DISPVECT) 
`
Right? Then maybe a little `private` member function would do, instead of `m_ConvertRAS`:
```

bool DoConvertRAS() const
{
  return (m_ConvertRASVectors && intent_code == NIFTI_INTENT_VECTOR) || (m_ConvertRASDisplacementVectors && intent_code == NIFTI_INTENT_DISPVECT);
}
```

Anyway, no problem, just a suggestion.

Reply:
We need to specify two behaviors: what to do if the file contains generic vector, and what to do if the file contains displacement vector.

We could just use one flag, but then overriding the default behavior would be more complicated: we would need to know what kind of data we read/write and change the flag accordingly. This is particularly inconvenient for reading, because we don't know the vector type of the file until we read the file header, so an extra pre-reading step could be necessary.

Reply:
OK, thank you for explaining. I was just wondering if we could do two flags, instead of three. But it's OK anyway!

Reply:
The proposed helper function would need to get the intent code and the source would be slightly different, depending on doing reading or writing (metadata dictionary or `m_NiftiImage->intent_code`). So, overall, I think the current implementation is a bit simpler. The third flag is private, so it does not add much complexity.